build.gradle

```diff
@@ -254,7 +258,14 @@ testClusters.integTest {
             return new RegularFile() {
                 @Override
                 File getAsFile() {
-                    return fileTree("src/test/resources/job-scheduler").getSingleFile()
+                    project.mkdir js_resource_folder
+                    ant.get(src: 'https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/'
+                            + opensearch_no_snapshot
+                            + '/latest/linux/x64/builds/opensearch/plugins/opensearch-job-scheduler-'
+                            + job_scheduler_no_snapshot + '.zip',
```

Comment:
Suggest to throw this into a constant since it's referenced twice

Reply:
added